GrandCentralBoard/Widgets/Watch/WatchWidget.swift

```diff
@@ -6,9 +6,7 @@
 import UIKit
 import GrandCentralBoardCore
 
-
-private let slideshowInterval = 10.0
-private let secondsInDay: NSTimeInterval = 3600
+private let secondsInAnHour: NSTimeInterval = 3600
```

Comment:
We agreed to go with the way SwiftLint likes (start nane from non-capital letter)